validator/roadmap.adoc

```diff
@@ -7,29 +7,35 @@ Hardy Ferentschik
 :toc-title: Releases
 
 Hibernate Validator is community driven and as such the roadmap constantly evolves to reflect the
-users needs.
+users needs and contributions.
+
 You can find a fine grained roadmap in our https://hibernate.atlassian.net/browse/HV[issue tracker],
 but this page is a good starting point to see where we are going.
 
-== Hibernate Validator 5.3
+== Hibernate Validator 6.0.x
 
-OSGi::
-Expose Validator as OSGi service (https://hibernate.atlassian.net/browse/HV-570[HV-570])
+Engine::
+Explore message interpolators receiving several locales (https://hibernate.atlassian.net/browse/HV-1436[HV-1436]) +
+Support custom field/getter selection strategy (https://hibernate.atlassian.net/browse/HV-1363[HV-1363])
 
 Annotation processor improvements::
-Taking into account XML configuration (https://hibernate.atlassian.net/browse/HV-308[HV-308]) +
-Verify group sequences (https://hibernate.atlassian.net/browse/HV-455[HV-455]) +
-Verify the validity of constraint declarations (e.g. min < max for @Size) (https://hibernate.atlassian.net/browse/HV-270[HV-270])
-
-Misc::
-Integration with JavaFX (https://hibernate.atlassian.net/browse/HV-653[HV-653]) +
-Introduce mechanism to order constraints (https://hibernate.atlassian.net/browse/HV-462[HV-462]) +
-Better integration with ORM, especially DDL constraint generation (https://hibernate.atlassian.net/browse/HV-830[HV-830]) +
-New range of Hibernate Validator specific constraints and potential dedicated
-constraint module (https://hibernate.atlassian.net/browse/HV-553[HV-553]) +
-Create standalone method interceptor using Byteman (https://hibernate.atlassian.net/browse/HV-432[HV-432])
-
-Internal::
-Provide ability to retrieve metadata from Jandex index (https://hibernate.atlassian.net/browse/HV-644[HV-644]) +
-Further unification of bean and method validation (https://hibernate.atlassian.net/browse/HV-467[HV-467]) +
-Switch to Gradle as build tool (https://hibernate.atlassian.net/browse/HV-641[HV-641])
+Taking into account new places where constraint annotations can be placed (https://hibernate.atlassian.net/browse/HV-879[HV-879]) +
+Taking into account XML configuration (https://hibernate.atlassian.net/browse/HV-308[HV-308])
+
+== Hibernate Validator 6.1
+
+Engine::
+Explore taking some distance from the reflection API (e.g. dealing with `Type` is rather confusing, replace some of the types with memory-friendly counterparts (`Method`?), do not rely on `TypeVariable`). Note that it might also help with the future usage of Jandex for annotation discovery.
+
+Programmatic API::
+Be able to define constraint on properties using lambdas (ie without constraint annotations)
+
+Java 9::
+Usage of method handles for accessing bean states (https://hibernate.atlassian.net/browse/HV-1226[HV-1226])
```

Comment:
s/method handles/var and method handles/

Also let's add

* "Explore usage of [ResourceBundleProvider](http://download.java.net/java/jdk9/docs/api/java/util/spi/ResourceBundleProvider.html)" for obtaining validation error messages from other modules
* Providing HV with JDK 9 module descriptors